src/templates/base/2019/index.html

```diff
@@ -84,16 +84,16 @@ <h3>{{ localizedChapterTitles[featured_chapter] }}</h3>
         </div>
         {% endif %}
       </div>
-      <a href="{{ url_for('chapter', year=2019, chapter=featured_chapter, lang=lang) }}" class="btn">
+      <a href="{{ url_for('chapter', year=year, chapter=featured_chapter, lang=lang) }}" class="btn">
```

Comment:
For things that are year-agnostic, should they be in the base/index.html template?

Reply:
Now we only have `templates/base/2019/` templates and no `templates/base/2020/` templates at all (since I got rid of the `2020/index.html` one in #1441 ), the `templates/base/2019/` basically are the year-agnostic ones.

There definitely is an argument to move these up a folder and remove the `templates/base/2019/` folder completely. But not as part of this PR. Will do that once there are no PRs open with base template changes.